go/codegen/experimental/codegen_couler.go

```diff
@@ -53,17 +52,14 @@ func GenerateCodeCouler(sqlProgram string, session *pb.Session) (string, error)
 	if err != nil {
 		return "", err
 	}
-	stepList := []*stepContext{}
+	stepList := make([]*stepContext, 0)
```

Comment:
For an empty slice, prefer `var stepList []*stepContext`.
c.f. https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices